tests/field/test_magneticfields.py

```diff
@@ -463,6 +513,11 @@ def test_Reiman(self):
         # Check that div(B)=0
         dB1 = Bfield.dB_by_dX()
         assert np.allclose(dB1[:, 0, 0]+dB1[:, 1, 1]+dB1[:, 2, 2], np.zeros((npoints)))
+        # Verify serialization works
+        field_json_str = json.dumps(Bfield, cls=MontyEncoder)
+        Bfield_regen = json.loads(field_json_str, cls=MontyDecoder)
+        Bfield_regen.set_points(points)
```

Comment:
It would be ideal if all the magnetic field classes saved the evaluation points. Presently only BiotSavart does right?

Reply:
All magnetic field classes now save the evaluation points when serialized.